test/unit/connection_string.test.ts

```diff
@@ -410,4 +410,46 @@ describe('Connection String', function () {
       expect(options).to.have.nested.property('credentials.source', 'thisShouldBeAuthSource');
     });
   });
+
+  describe('feature flags', () => {
+    it('should map known symbols onto options', () => {
+      const randomFlag = Symbol();
+      const client = new MongoClient('mongodb://iLoveJavaScript', { [randomFlag]: 23n });
+      expect(client.s.options).to.not.have.property(randomFlag);
+    });
+
+    it('should map all default symbols and values onto options', () => {
+      const client = new MongoClient('mongodb://iLoveJavaScript');
+      expect(FEATURE_FLAGS.size).to.be.greaterThanOrEqual(1);
```

Comment:
Are they expectations about the size of feature flags necessary?  FEATURE_FLAGS is a const set

Reply:
This test isn't even doing anything of worth anymore now that we removed the defaulting so I removed it